app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -14,6 +14,17 @@ def google_oauth2
     end
   end
 
+  def facebook
```

Comment:
We should test this new method!

Reply:
You'll want to add the test in `spec/controllers/omniauth_callbacks_controller_spec.rb`. The `GET #google_oauth2` describe block should give you hints on how to test the Facebook auth.

Reply:
This test will be helpful for uncovering potential bugs @bmwachajr has brought up below :)